examples/ssx-test-token-gated/README.md

```diff
@@ -0,0 +1,338 @@
+# SSX ENS Token Gated + RainbowKit
+
+This example will show developers how to enable token-gated access in their dapp with SSX. Additionally, it will show a developer how to integrate and use RainbowKit in an SSX-powered dapp.
+
+## Create the dapp
+
+The initial setup will be done using SSX's dapp creation tool. Type the following in your terminal to get started:
+
+```bash
+yarn create @spruceid/ssx-dapp token-gated-example
+```
+
+For this example we will be using the following options in the setup tool:
+
+- Typescript
+- Leave the other options empty
+
+## Setup Alchemy Account
+
+An Alchemy account and API key will be necessary since alchemy-sdk is used to resolve tokens owned by an account.
+To configure the key in the project create a `.env.development` file and add the key like the following:
```

Comment:
What folder will this be under? Define the path. I created it under src so this was my path `token-gated-example>src>.env.development`

Reply:
`.env.example` is already provided.